string[] actuator_ids
---
string message
